extensions/wikia/ImageServing/drivers/ImageServingDriverBase.class.php

```diff
@@ -170,7 +159,7 @@ protected function getFileByName( $fileName ) {
 	protected function storeInCache( $articleImageIndex ) {
 		// store images for each article separately
 		foreach ( $articleImageIndex as $articleId => $imageIndex ) {
-			$this->memc->set( $this->makeKey( $articleId, $this->getRevId( $articleId ) ), $imageIndex, 3600 );
```

Comment:
Since this has revId in it, could this TTL be longer than 3600?

Reply:
Yeah, we could try that out as follow-up :)